examples/dti_prediction/README.md

```diff
@@ -0,0 +1,40 @@
+# Drug-Target Interaction: DeepDTA
```

Comment:
You name this example as `dti_prediction`, which is different from our current naming convention for Examples: data_method. Also dti is not very readable for outsiders. Can you try to use a name following the other examples?

Reply:
If you are making this change, try to do it as a single commit and do not mix with other commits. This will make the history cleaner and more readable (though longer, see the contribution guidance on small commits).